documentation/sphinx/source/api-python.rst

```diff
@@ -325,12 +325,6 @@ A |database-blurb1| |database-blurb2|
 .. |sync-read| replace:: This read is fully synchronous.
 .. |sync-write| replace:: This change will be committed immediately, and is fully synchronous.
 
-.. method:: Database.allocate_tenant(tenant_name):
-
-    Creates a new tenant in the cluster. |sync-write|
-
-    The tenant name can be either a byte string or a tuple and cannot start with the ``\xff`` byte. If a tuple is provided, the tuple will be packed using the tuple layer to generate the byte string tenant name.
-
 .. method:: Database.delete_tenant(tenant_name):
```

Comment:
Should the documentation for Database.delete_tenant be removed too?

Reply:
Yes, good catch.